packages/earljs/src/isEqual/index.ts

```diff
@@ -0,0 +1,5 @@
+export * from './EqualityOptions'
+export { CanonicalType, getCanonicalType } from './getCanonicalType'
+export { getKeys } from './getKeys'
+export { isEqual } from './isEqual'
+export { isEqualUnknown } from './isEqualUnknown'
```

Comment:
Do we want to export this? Isn't it an internal implementation detail of the algorithm? (It's used on in `isEqual.ts` file as far as I see.)

Reply:
Yes, this shouldn't be exported. It was using it in `format` at some point, forgot to remove the export after refactoring.